index.html

```diff
@@ -17,80 +17,129 @@
   <body>
     <section class="home-page">
       <header>
-        
         <ul id="headerList">
           <li class="mobile menu_">
-            <a><img src="img\mobile-img\menu-outline.svg" id="menuButton" /></a>
+            <a
+              ><img
+                src="img/mobile-img/menu-outline.svg"
+                id="menuButton"
+                alt="menu-button"
+            /></a>
+          </li>
+          <li class="Desktop facebook">
+            <a><p>f</p></a>
+          </li>
+          <li class="Desktop">
+            <a
+              ><img
+                class="twitter"
+                src="img/Desktop-img/twitter.png"
+                alt="twiiter logo"
+            /></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">English</p></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">My page</p></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">Logout</p></a>
           </li>
-          <li class="Desktop"><a></a></li>
-          <li class="Desktop"><a></a></li>
-          <li class="Desktop"><a></a></li>
-          <li class="Desktop"><a></a></li>
         </ul>
-        <div  class="menu hidden">
+        <div class="menu mobile hidden">
+          <img src="img/mobile-img/cancel.png" alt="cancel" class="cancel" />
           <ul>
             <li class="home"><a>Home</a></li>
             <li class="about"><a>About</a></li>
-            <li><a class="buyTicket">BUY TICKET</a></li>
+            <li><a class="buyTicket">Join us</a></li>
+          </ul>
+        </div>
+        <div class="menu Desktop">
+          <ul class="menuDesk">
+            <li>
+              <a
+                ><img
+                  src="img/poth-version-img/large.png"
+                  class="LogoDesktop"
+                  alt="Logo for desktop"
+              /></a>
+            </li>
+            <li class="home"><a>Home</a></li>
+            <li class="about"><a>About</a></li>
+            <li><a class="buyTicket">Join us</a></li>
           </ul>
         </div>
       </header>
+
       <div class="container">
-      <p class="display-1">"Hello! Sharing world"</p>
-      <h1 class="display-1 fw-bold">
-        Global Designers <br />summit <img class="mask" /><span
-          class="display-1 fw-bold"
-          >2021</span
-        >
-      </h1>
-      <div class="text-container">
-        <p>
-          A joufyl celebration believing in the value of opennes and sharing,
-          creating a positive change with people from over 80 countries is
-          taking place in October, in Korea.
+        <p class="display-1">"Hello! Sharing world"</p>
```

Comment:
**Kindly provide the requested design by the template:**

_**Template: **_
<img width="896" alt="Screen Shot 2021-11-19 at 6 19 30" src="https://user-images.githubusercontent.com/22258274/142622317-91c92926-cb40-41d4-92f3-9459b8f51abe.png">
<img width="1080" alt="Screen Shot 2021-11-19 at 6 18 50" src="https://user-images.githubusercontent.com/22258274/142622324-c21272d3-b182-4c40-a595-61e07904becf.png">
<img width="1080" alt="Screen Shot 2021-11-19 at 6 19 05" src="https://user-images.githubusercontent.com/22258274/142622326-7341e86a-34cc-468f-8c0a-835ba762ffea.png">

#
_**- Yours:**_
- Fix your fonts and style:
<img width="1440" alt="Screen Shot 2021-11-19 at 6 21 39" src="https://user-images.githubusercontent.com/22258274/142622536-ae60fd33-c3c2-4e09-8e67-cd577f8b6e0f.png">

- Fix image sizes:
<img width="1440" alt="Screen Shot 2021-11-19 at 6 25 02" src="https://user-images.githubusercontent.com/22258274/142622539-60b7eb42-b562-471c-94ec-8486d2f293b1.png">
- Align your footer items:
<img width="1440" alt="Screen Shot 2021-11-19 at 6 25 25" src="https://user-images.githubusercontent.com/22258274/142622543-aad1fc8f-52fe-4aac-a3fe-96f77fb382eb.png">


## Also kindly check your ``about.html``:

<img width="1440" alt="Screen Shot 2021-11-19 at 6 28 43" src="https://user-images.githubusercontent.com/22258274/142622915-bdd8c876-7a69-463a-8844-11bd13916186.png">